### PR TITLE
Compression detection added

### PR DESCRIPTION
**Pull request by jensg-st**
Signed-off-by: Jens Gerke <jens.gerke@sisa-tech.com>

### CODE REVIEW

**Comment by jensg-st** [comment]
Fixed whitespaces and squashed commits.

**Comment by stgraber** [comment]
Note that you could have squashed the commits and updated your pull request without closing it by just doing a git push --force. That would have conserved the existing discussion.

**Comment by jensg-st** [comment]
Sorry. More a Mercurial guy so far. Will improve I hope and get my git-fu sorted :)

File: lxd/images.go [hunk 1]
```diff
@@ -151,29 +252,29 @@ func extractTar(fname string) (int, error) {
...
+	//			return 0, nil
+	//			//return 0, err
+	//		}
+	//		if hdr.Name != "metadata.yaml" {
+	//			continue
+	//		}
+	//		//tr.Read()
+	//		// find architecture line
+	//		break
+	//	}
```
Comment (tych0) [inline lxd/images.go]: Can we just axe this commented out code (and the stuff below)? I don't think there's any reason to keep it.

**Comment by jensg-st** [comment]
Next try. Sorry for having issues with Git. Maybe I have done it right this time...

**Comment by tych0** [comment]
Awesome, thanks!

**Pull request merged**